.github/workflows/release.yml

```diff
@@ -75,13 +85,23 @@ jobs:
         continue-on-error: true
         working-directory: marine-js/npm-package
 
+      - name: Commit version bump
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "chore: Bump marine-js version to ${{ steps.version.outputs.version }}"
+          branch: ${{ fromJson(needs.release-please.outputs.pr).headBranchName }}
+          commit_user_name: fluencebot
+          commit_user_email: devops@fluence.one
+          commit_author: fluencebot <devops@fluence.one>
+
+        # workaround for marine depending on itself messing with lock file
       - name: Update Cargo.lock
         run: cargo update -p marine -p mrepl
```

Comment:
Lets use `cargo update --workspace`, docks say that it is correct way to update lockfile after bumping versions https://doc.rust-lang.org/cargo/commands/cargo-update.html#update-options